src/Publish/Microsoft.NET.Sdk.Publish.Tasks/WebConfigTransform.cs

```diff
@@ -1,10 +1,9 @@
-﻿using System;
+﻿using Microsoft.NET.Sdk.Publish.Tasks.Properties;
```

Comment:
system first?

Reply:
For sure. New VS installation messed up my setting. Will change it.